README.md

```diff
@@ -74,44 +51,39 @@ Kubernetes directly.
 **Configuration 3:**
 ![screenshot](./images/gke-deployment.png)
 
-A simple Python Flask web application (Prime-flask) was created for this
-demonstration which contains two endpoints:
+A simple Python [Flask](http://flask.pocoo.org/) web application (`Prime-flask`) was created for this demonstration which contains two endpoints:
 
 `http://[ip]:8080/factorial/` and
 
 `http://[ip]:8080/prime/`
 
-examples would look like:
+Examples of use would look like:
 
 ```console
-curl http://[ip]:8080/prime/10
+curl http://35.227.149.80:8080/prime/10
```

Comment:
unless there is a reason to specify a static IP here, I would suggest a place holder or var. ie: `curl "http://${ip}:8080/prime/10"

Reply:
So, I disagree. I think console output should be representative. When example commands are given I do agree, however.

Reply:
I'm picking up what you're putting down. I'm cool with this. but in this case, please use real bash syntax in 56 and 58 for vars (same comment throughout)

Reply:
Per off-line conversation, we're agreed on `<ip>`